gr-uhd/grc/uhd_rfnoc_duc.block.yml

```diff
@@ -7,7 +7,6 @@ templates:
   make: |-
     uhd.rfnoc_duc(
         self.rfnoc_graph,
-        ${num_chans},
```

Comment:
Was this an error in the original yml? I don't see a corresponding code change.

Reply:
Yes, it was. I should probably tag this as a bug fix.